### DISCUSSION TITLE
Another person saying thanks!

### DISCUSSION THREAD

**Discussion by ZebulonGlitch** [tranxuanthang/lrcget#56] [Ideas]
I too am here to say thanks!  I'll also be sending to your paypal as a more REAL way of showing gratitude.  ;-)

The software is still rough as hell and hard to use, but holy cow, it still saves an incredible amount of time!  Please please please, improve the list view so that it supports a work flow.  I am a huge CD collector and have over 800 albums i've bought and ripped over the years making around 10,000 mp3's and flac's.  I never dreamed I'd maybe have a chance to add lyrics to them all.  It's so nice in Kodi to read the sync'd lyrics on the screen when enjoying music.  :)

- What I mean by workflow above:  The app currently is missing basic day to day features like filtering the list, searching the list, and marking instrumental tracks so that they maybe get a standard "No Vocals" or "instrumental" lrc record.  Users need to see what tracks need attention, filter by problems, hide good tracks, etc...  Once it has these basics, you can truly call it a "lyric manager!"  :D :D :D
- I also created and fixed a few LRC files using your timing editor - just for fun.  It's buggy but works!  (once you figure out it's quirks)  
- I lastly discovered that many of the synced lyrics from the community are less than perfect when used on Exact Audio and musicbrainzed original album rips.  Seems to need a rating system for the LRC files that uses maybe the MusicBrainz ID of each song as an identification fingerprint.  I tested the timing on a batch of community LRC files, some were good, some were bad, and I fixed a few and submitted them.  I would like it if that "quality" information was also uploaded to the community so that the quality of the LRC data improves over time.

Message me if you're interested in little improvement tips. Otherwise I suspect you already know what needs to be done.  :)

Please keep up the good work!

**Comment by tranxuanthang** [comment]
Thank you so much! Having some trouble with life lately but your support and your message came in and made me feel so much better.

Some of your feedbacks are already the priority for the next update: instrumental marking (https://github.com/tranxuanthang/lrcget/issues/36), sorting, filtering and searching the library. For the rating system, I will need more time to think about it, and a major change in design of LRCLIB might be needed (see https://github.com/tranxuanthang/lrcget/issues/42).

I always love to hear more suggestions from you or anyone, and also curious about what trouble you are facing with the lyrics editor and how to make it better.
